proxy_lua.c

```diff
@@ -1795,6 +1684,12 @@ int proxy_register_libs(void *ctx, LIBEVENT_THREAD *t, void *state) {
         {NULL, NULL}
     };
 
+    const struct luaL_Reg mcplib_inspector_m[] = {
```

Comment:
maybe make a reqinspector/resinspector as different meta tables? I don't think this helps much though.

Reply:
meh maybe later. should do an actual asm comparison + branch miss count to see if it does anything. odds are pretty good it wouldn't.